src/screens/IndexScreen.js

```diff
@@ -145,6 +148,14 @@ export const IndexScreen = ({ navigation, route }) => {
       [QUERY_TYPES.EVENT_RECORDS]: showEventsFilter,
       [QUERY_TYPES.NEWS_ITEMS]: showNewsFilter
     }[query];
+  const htmlContentName =
+    (query === QUERY_TYPES.POINTS_OF_INTEREST && poiListIntro[queryVariables.category]) || '';
+
+  const { data: htmlContent } = useStaticContent({
+    name: htmlContentName,
+    type: 'html',
+    refreshTimeKey: `${query}-${queryVariables.category}`
+  });
```

Comment:
could you explain what this part of the code does since I'm not very familiar with queries yet?